lib/lightning_web/live/credential_live/credential_form_component.ex

```diff
@@ -432,10 +432,9 @@ defmodule LightningWeb.CredentialLive.CredentialFormComponent do
             <span class="font-bold"><.modal_title action={@action} /></span>
             <button
               id={"close-credential-modal-form-#{@credential.id || "new"}"}
-              phx-click="close_modal"
-              phx-target={@myself}
+              phx-click={JS.navigate(@return_to)}
```

Comment:
Hey @jyeshe I don't have a lot of background context for this work but I was wondering why don't we do this `JS.navigate` thing for the other **X** close buttons in the page ? The OAuth one for example ?

Reply:
Done